StandardLibrary/test/TestComputeSetToOrderedSequenceCharLess.dfy

```diff
@@ -92,9 +92,20 @@ module TestComputeSetToOrderedSequenceCharLess {
     var output := ComputeSetToOrderedSequence(a, CharLess);
     var output2 := ComputeSetToOrderedSequence2(a, CharLess);
     var expected := ["&", "Љ", "ᝀ", "𐀂", "𐐷", "🂡", "｡"];
-    // This is the pure logographic order
+    // This is the pure logographic order,
     // however this function is used in the DB-ESDK
-    // to canonicalized sets and needs to remain the same.
+    // to canonicalize sets, and needs to remain the same.
+    // The expected ordering for strings is "UTF-16 Binary Order",
+    // where characters are converted to their UTF-16 big endian representations
+    // and compared starting at the first bytes
+    // (or, UTF-16 little endian and compared starting at their last bytes).
+    // Note that non-string characters should NOT be UTF encoded.
```

Comment:
I cannot find line 102 in the spec, and it confuses me.

What is a non-string character?

Reply:
This is inaccurate phrasing. Let me clarify.